pdm.lock

```diff
@@ -1527,27 +1568,27 @@ files = [
 
 [[package]]
 name = "ruff"
-version = "0.0.291"
+version = "0.1.3"
```

Comment:
Ugh, looks like we didn't completely update our `pdm.lock` file when we changed to the ruff formatter. I'll create a separate PR to change this so that yours doesn't include those changes as well 👍.

Reply:
https://github.com/pydantic/pydantic/pull/8005 should fix this, then you can pull that in + make sure you're not pushing changes to `pdm.lock` 😄

Reply:
I'll update that tonight.